graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -326,25 +471,27 @@ def enter(
             renamed_node = self._rename_name_and_add_to_record(cast(RenameTypes, node))
             if renamed_node is node:  # Name unchanged, continue traversal
                 return IDLE
-            else:  # Name changed, return new node, `visit` will make shallow copies along path
+            else:
+                # Name changed or suppressed, return new node, `visit` will make shallow copies
+                # along path
                 return renamed_node
         else:
             # All Node types should've been taken care of, this line should never be reached
             raise AssertionError('Unreachable code reached. Missed type: "{}"'.format(node_type))
 
 
 class RenameQueryTypeFieldsVisitor(Visitor):
```

Comment:
Consider updating this name, since it no longer just renames and can now also remove.

Reply:
I looked at the helper function here and we can actually get away with just returning the helper's output (after a small change), so no need to actually come up with a good variable name here :)